typescript/test/utils/mock-bridge.ts

```diff
@@ -314,4 +326,40 @@ export class MockBridge implements Bridge {
   walletRegistry(): Promise<WalletRegistry> {
     throw new Error("not implemented")
   }
+
+  buildRedemptionData(
+    redeemer: Identifier,
+    walletPublicKey: string,
+    mainUtxo: UnspentTransactionOutput,
+    redeemerOutputScript: string
+  ): Hex {
+    this._buildRedemptionDataLog.push({
+      redeemer,
+      walletPublicKey,
+      mainUtxo,
+      redeemerOutputScript,
+    })
+
+    // Convert the output script to raw bytes buffer.
+    const rawRedeemerOutputScript = Buffer.from(redeemerOutputScript, "hex")
+    // Prefix the output script bytes buffer with 0x and its own length.
+    const prefixedRawRedeemerOutputScript = `0x${Buffer.concat([
+      Buffer.from([rawRedeemerOutputScript.length]),
+      rawRedeemerOutputScript,
+    ]).toString("hex")}`
```

Comment:
Looks like we could create a reusable fn for this because we use it in unit tests and in Ethereum Bridge implementation. But I'm not sure which place is better- `ethereum-helpers.ts` or `bitcoin.ts? cc @lukasz-zimnoch @nkuba what do you think guys?